iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h

```diff
@@ -173,6 +173,17 @@ struct IECExpressionsCacheProvider
     ECExpressionsCache& Get(Utf8CP rulesetId) {return _Get(rulesetId);}
 };
 
+//===================================================================================
+// @bsiclass
+//===================================================================================
+enum class ECExpressionEvaluationResult
+    {
+    EVALUATION_Success,
+    EVALUATION_ParseError,
+    EVALUATION_EvaluationError,
+    EVALUATION_InvalidECValueError

```

Comment:
The `EVALUATION_` prefix is not needed for enum class

Reply:
Removed